### PR TITLE
Update SC-23

### PR DESCRIPTION
**Pull request by kevindoingbizas**
https://issues.redhat.com/browse/CMP-408?filter=12344992

### CODE REVIEW

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 1]
```diff
@@ -1024,10 +1024,16 @@
         OpenShift uses various TLS algorithms and ciphers to protect
         the authenticity of communications sessions.
 
-        A complete control response is planned. Engineering progress can be
-        tracked via:
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: Does it make sense to reword or delete the original placeholder text entirely?

Comment (shawndwells) [reply openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: (leaning towards blowing it away)

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 2]
```diff
@@ -1024,10 +1024,16 @@
...
+        enhancement, which ensures that only HTTPS traffic is
+        allowed on the host. Any HTTP requests are dropped by default.
+        This is useful for ensuring secure interactions with websites,
+        or to offer a secure application for the user’s benefit.
+
+        https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
 
-        https://issues.redhat.com/browse/CMP-408
+        In addition, all web based endpoints are configured
+        to use HTTPS by default.
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: What setting in what file can be evaluated to make sure the default was not changed?

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml [hunk 3]
```diff
@@ -1024,10 +1024,16 @@
...
 
-        A complete control response is planned. Engineering progress can be
-        tracked via:
+        HTTP Strict Transport Security (HSTS) policy is a security
+        enhancement, which ensures that only HTTPS traffic is
+        allowed on the host. Any HTTP requests are dropped by default.
+        This is useful for ensuring secure interactions with websites,
+        or to offer a secure application for the user’s benefit.
+
+        https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration
```
Comment (shawndwells) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: In addition to HSTS, are there other settings that may impact the authenticity of communication sessions? For example:
- Server side PKI certs
- User PKI

**Comment by shawndwells** [comment]
@kevindoingbizas excellent formatting, tone, and reference to docs! A few clarifying questions/comments.

File: openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml (hunk 3 above)
Comment (shawndwells) [inline openshift-container-platform-4/policies/SC-Systems_and_Communications_Protection/component.yaml]: To help make the docs evergreen, should ``latest`` be used in the URL?

https://docs.openshift.com/container-platform/4.3/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration

vs

https://docs.openshift.com/container-platform/latest/networking/routes/route-configuration.html#nw-enabling-hsts_route-configuration

**Comment by redhatrises** [comment]
The comments in this PR haven't been addressed or resolved in a few months. This PR looks frozen. Closing. Please re-open this PR when the comments are addressed appropriately.

**Pull request closed without merging**